scenarioo-client/prepareProtractor.js

```diff
@@ -28,7 +32,7 @@ function prepareProtractorForE2ETests() {
         // this is also needed if scenarioo documenztation is not enabled
         var scenarioo = require('scenarioo-js');
         scenarioo.setupFluentDsl();
-        require('./test/protractorE2E/labelDefinitions');
+        require('./test/protractorE2E/labelDefinitions.js');
```

Comment:
the import should not be necessary to know whether it is a js or not

Reply:
concerning this not working properly with typescript: please open an issue on scenariooJS repo, might be something missing in the typings. eat your own dog food rules ;-)